settings.yml

```diff
@@ -41,6 +41,11 @@ ma:
   - chamber: lower
     district: 11th Plymouth
     vacant_until: 2023-01-14
+ny:
+  vacancies:
```

Comment:
@bfossen-ce lester chang is in lower-49 so it shouldn't be represented as vacant.